openslides_backend/action/motion/update.py

```diff
@@ -33,5 +35,33 @@ class MotionUpdate(UpdateAction, AmendmentParagraphsMixin):
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         instance["last_modified"] = round(time.time())
+        if instance.get("text"):
+            motion = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]), ["text"]
+            )
```

Comment:
Please bundle all requests into one that only gets executed if any of the conditions apply.

Reply:
Bundle the motion get calls into one. Add a condition so, it is only called, if needed.